dora/src/bytecode/generator.rs

```diff
@@ -805,7 +813,9 @@ impl<'a, 'ast> AstBytecodeGen<'a, 'ast> {
                     self.emit_invoke_direct(return_type, return_reg, fct_def_id, num_args);
                 } else if fct.is_virtual() {
                     self.emit_invoke_virtual(return_type, return_reg, fct_def_id, num_args);
-                } else if arg_bytecode_types[0] != BytecodeType::Ptr {
+                } else if arg_bytecode_types.is_empty()
```

Comment:
This change is probably also needed because of using `CallType::Method`.

Reply:
Yes, removed after adding `ModuleMethod`.